### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by Toxu-ru** [Meneame/meneame.net#5]
@eusonlito Where is the installation instructions? In the wiki section all links are broken. Where I can learn in detail how to install it.

**Comment by osvid** [comment]
Hi Toxi, afaik there are no installation notes. I did set up a running clone of mnm at https://poemame.com a time aho. I'm right now in the process of updating my development environment so I'd try to generate and share an updated sort of instructions. 

@eusonlito, is mnm compatible with Android PHP7.x?

**Comment by Toxu-ru** [comment]
@osvid Thanks for the reply. It is compatible with PHP 7.x. There are some problems with mysql 5.7, but they are solved in the usual way. 

I managed to put it up and tweak it, though there are some questions. Just without documentation, installation may not be as easy.

**Comment by eusonlito** [comment]
Yes, it's compatible with PHP 7.2 and almost 7.3 (Haanga template engine is not compatible).

MySQL 5.7 is supported with `sql-mode = ''`

**Comment by eusonlito** [comment]
@osvid great job with https://poemame.com/ :)

**Comment by Toxu-ru** [comment]
@eusonlito  Maybe it makes sense to open engine support on your site (create space, group, stream)? So someone could ask questions. I just noticed that very few people know this wonderful scenario, which is strange.

**Comment by osvid** [comment]
@eusonlito Thanks! I'm in the process of revamping it, to merge some of your recent commits and add some new functionalities requested by the users' community. I'll keep you updated!

**Comment by RafaelOsuna12** [comment]
Hello friends.

I'm crazy looking for installation information and can not find anywhere, I followed some internet documents but they are too old and do not work.

They would help me a little with the installation of meneame, the file that I generate -local.php generates me error 500:

<? php

$ globals ['site_name'] = '';
$ globals ['db_server'] = '';
$ globals ['db_name'] = '';
$ globals ['db_user'] = '';
$ globals ['db_password'] = '';

?>

I found it on the internet but I think it's not the right one.

Thank you very much for your attention friends!

**Comment by evgip** [comment]
@RafaelOsuna12  You have provided little information about the error. What do the logs say?

**Comment by RafaelOsuna12** [comment]
Hello @Toxu-ru 
I can not connect the database, where do I find the records?
regards

**Comment by evgip** [comment]
This is the config file:
https://github.com/Meneame/meneame.net/blob/master/www/config.php#L78

**Comment by RafaelOsuna12** [comment]
Thanks for the reply.

What happens is that just in that configuration file made the changes to the credentials of the database. I import the bases .sql (meneame, categories, subs) and when I try to enter the site I am loaded with error 500.

**Comment by evgip** [comment]
@RafaelOsuna12  Try this archive, I'm a bit corrected for the Apache.
https://github.com/SugataRu/sugata.ru
However, many templates have been reworked + some functions, so it will not work with the original version.
